### DISCUSSION TITLE
hexo-theme-next 如何修改默认显示文字

### DISCUSSION THREAD

**Discussion by FHR2048** [walinejs/waline#1923] [Q&A] [answered]
[https://waline.js.org/cookbook/customize/locale.html](https://waline.js.org/cookbook/customize/locale.html)
上面链接中提供了更改自定义语言的配置项
在hexo-theme-next中评论框默认文字更改选项应该就是：
waline:
......
  placeholder: xxxxxxxx
.....
但这么更改是无效的,正确的修改方式是什么呢？

**Comment by lizheming** [comment, marked as answer]
https://github.com/orgs/walinejs/discussions/1922 是 locale.placeholder 不是 placeholder
